landoapi/commit_message.py

```diff
@@ -72,7 +72,7 @@ def format_commit_message(
     approvals: List[str],
     summary: str,
     revision_url: str,
-    flags: List[str] = None,
+    flags: Optional[List[str]] = None,
```

Comment:
I personally prefer the `List[str] | None` notation to using `Optional`, here and elsewhere. Personal preference only.